src/chatdlg.cpp

```diff
@@ -132,7 +132,7 @@ void CChatDlg::OnAnchorClicked ( const QUrl& Url )
     {
         if ( QMessageBox::question ( this,
                                      APP_NAME,
-                                     tr ( "Do you want to open the link" ) + " <b>" + Url.toString() + "</b> " + tr ( "in an external browser?" ),
+                                     tr ( "Do you want to open the link '%1' in your browser?" ).arg ( "<b>" + Url.toString() + "</b>" ),
```

Comment:
I hadn't realised `tr()` returns a QString that can then have `arg()` applied.  This is a nice idiom that could be used further.

Reply:
This patchset contains all the strings I found while doing the Simplified Chinese translation that concatenates use `+` instead of a formatted string with `arg()` (since it causes trouble for translating). I'm not sure if I missed some other places but most major strings I found while translating are covered in this PR.